### PR TITLE
Wallet fixup

### PR DESCRIPTION
**Pull request by Mingling94**
On top of #142 
Here's what the plugin looks like half screen and whole screen:
sia_044
sia_043

### CODE REVIEW

**Comment by lukechampine** [comment]
The red seems kinda bright. Is there a corresponding green for when you received money?

**Comment by Mingling94** [comment]
There is, they are all-red or all-green, meaning hex-values #FF0000 or #00FF00, I can tone them down a bit to like #aa0000 and #00aa00 if you'd like.

**Comment by pmknutsen** [comment]
Why remove the copy-to-clipboard shortcut in Addresses? I find it very convenient.

**Comment by pmknutsen** [comment]
And, was the Create Address removed as well, or moved elsewhere?

**Comment by Mingling94** [comment]
@pmknutsen I enabled a right-click context-menu a while ago with the whole UI that allows copy-paste, but I could add that back in. Create Address is the "+" button in the lower right.

**Comment by DavidVorick** [comment]
I am in favor of bringing back the copy-to-clipboard button

**Comment by Mingling94** [comment]
Copy-to-clipboard button is back!
sia_045

**Comment by Mingling94** [comment]
@lukechampine @DavidVorick boop

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "doc": "jsdoc -c .jsdocrc",
     "sia-repo": "go get -u github.com/NebulousLabs/Sia/...",
     "sia-import": "(test -d Sia || mkdir -p Sia) && (cp $GOPATH/bin/siad Sia & cp $GOPATH/bin/siac Sia)",
-    "sia-kill": "killall siad; while pgrep siad; do sleep .5; done;",
+    "sia-kill": "killall -s SIGKILL siad; while pgrep siad; do sleep .5; done;",
```
Comment (lukechampine) [inline package.json]: Don't send `SIGKILL`. `SIGTERM` gives siad a chance to shutdown cleanly. Sending `SIGKILL` could cause corruption.

File: plugins/Wallet/css/lists.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+/* Transactions list */
+.transaction > * {
+	overflow: hidden;
+}
+.currency.positive {
+	color: #00FF00;
+}
+.currency.negative {
+	color: #FF0000;
+}
```
Comment (lukechampine) [inline plugins/Wallet/css/lists.css:48]: yeah, these are too bold. Tone them down a bit.

File: plugins/Wallet/js/transactions.js [hunk 3]
```diff
@@ -92,13 +92,13 @@ $('#transaction-page').on('input', updateTransactionPage);
...
-	       (criteria.maxValue  === undefined || txn.value                 <= criteria.maxValue)  &&
-	       (criteria.minValue  === undefined || txn.value                 >= criteria.minValue)  &&
-	       (criteria.currency  === undefined || criteria.currency.indexOf(txn.currency) !== -1)  &&
-	       (criteria.inputs    === true      || txn.value                 >= 0)                  &&
-	       (criteria.outputs   === true      || txn.value                 <= 0);
+	return (criteria.endTime   === null || txn.confirmationtimestamp <= criteria.endTime)   &&
+	       (criteria.startTime === null || txn.confirmationtimestamp >= criteria.startTime) &&
+	       (criteria.maxValue  === null || txn.value                 <= criteria.maxValue)  &&
+	       (criteria.minValue  === null || txn.value                 >= criteria.minValue)  &&
+	       (criteria.currency  === null || criteria.currency.indexOf(txn.currency) !== -1)  &&
```
Comment (lukechampine) [inline plugins/Wallet/js/transactions.js:99]: what's the advantage of switching to `null` here? I assume it protects against using non-existent fields, e.g. `criteria.foo`?

Comment (Mingling94) [inline plugins/Wallet/js/transactions.js:99]: No advantage, there are only slight differences to them. It just felt wrong to initialize something, or `define` it, as `undefined`.

Rather a null criteria just means it's relaxed and won't limit the transaction search.

**Comment by lukechampine** [comment]
for future reference, better than amending here would be to specifically `git revert c219021` to make your intentions clear.

**Comment by DavidVorick** [comment]
The eyeglass is too close to the addresses, it looks unnatural

**Comment by Mingling94** [comment]
boop @DavidVorick @lukechampine

**Pull request merged**